assets/x8.xml

```diff
@@ -0,0 +1 @@
+<!-- Replace with final version of our fixed-wing model -->
```

Comment:
@yekime update with current fixed wing model
and if there are any other asset files related to sim/unreal might be useful to store in this folder